include/mbgl/style/expression/parsing_context.hpp

```diff
@@ -76,22 +76,22 @@ enum class TypeAnnotationOption {
 
 class ParsingContext {
 public:
-    ParsingContext()
+    ParsingContext() noexcept
```

Comment:
`make_shared` needs to allocate so this cannot be made `noexcept`

Reply:
yeah... no idea what I was thinking there